oat-sa/extension-tao-itemqti
# FIX/AUT-2088 Fix horizontal size choices

1/2
Releated to [AUT-2088](https://oat-sa.atlassian.net/browse/AUT-2088)

**Description:**
Add handler to remove predefined height when switching from vertical to horizontal list in choices interaction.

**Changes:**
* Added reset height method that is setting the height to auto for previously set adapted heigh

**How to check:**
* Create a new item
* Add choices Interaction
* Add an inline image to the second option
* Check between vertical/horizontal mode
* Should not take more space anymore in horizontal mode

**Requires:**
* Require [tao-core](https://github.com/oat-sa/tao-core-sdk-fe/pull/144)
